src/liquid/components/ld-checkbox/readme.md

```diff
@@ -182,11 +196,13 @@ If the `indeterminate` attribute is present on the `ld-checkbox` component, the
 {% example '{ "background": "light" }' %}
 <ld-checkbox indeterminate></ld-checkbox>
 
-<ld-checkbox indeterminate disabled></ld-checkbox>
```

Comment:
Seems like you removed the `indeterminate disabled` component from the example. Is that intended?

Reply:
Yes, this is intended. Disabled is explained above and I realized there is no need to display each following example with a disabled state. It also is more user friendly for copy pasting examples and looks less cluttered.